### ISSUE TITLE
Is there a methods table missing here?

### ISSUE THREAD

**Issue by Jimnar** [microsoftgraph/microsoft-graph-docs#21862]
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: f939bcee-8a57-8f6f-9110-5df84da79dcd
* Version Independent ID: f705a342-f0fc-c6fe-b8da-e758d1ddffd9
* Content: [agreementFileLocalization resource type - Microsoft Graph v1.0](https://learn.microsoft.com/en-us/graph/api/resources/agreementfilelocalization?view=graph-rest-1.0)
* Content Source: [api-reference/v1.0/resources/agreementfilelocalization.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/v1.0/resources/agreementfilelocalization.md)
* Product: **governance**
* Technology: **microsoft-graph**
* GitHub Login: @raprakasMSFT
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by lramosvea** [comment]
@FaithOmbongi FYI

**Comment by AlexFilipin** [comment]
Fair point, there are not many methods but in the table of context you can find:

- https://learn.microsoft.com/en-us/graph/api/agreementfile-get?view=graph-rest-1.0&tabs=http
- https://learn.microsoft.com/en-us/graph/api/agreement-post-files?view=graph-rest-1.0&tabs=http
- https://learn.microsoft.com/en-us/graph/api/agreement-list-files?view=graph-rest-1.0&tabs=http

@FaithOmbongi can you take a look at updating the methods or should I take that action item?

**Comment by FaithOmbongi** [comment]
Hi @AlexFilipin - Yep, you're right on both counts.

**Comment by AlexFilipin** [comment]
Thanks @FaithOmbongi , reopening so we can fix holistically - let me know if are going to fix it or if I would take action. :)

**Comment by FaithOmbongi** [comment]
Hi @AlexFilipin - Yes, please raise the docs PR. Thanks.
